Sources/AppsFlyerInstance.swift

```diff
@@ -61,31 +72,78 @@ public class AppsFlyerInstance: NSObject, AppsFlyerCommand {
+            appsFlyer.disableAppleAdsAttribution = disableAppleAdsAttribution
+        }
+        if let disableCollectASA = settings[AppsFlyerConstants.Configuration.disableCollectASA.rawValue] as? Bool {
+            appsFlyer.disableCollectASA = disableCollectASA
+        }
+        if let minTimeBetweenSessions = settings[AppsFlyerConstants.Configuration.minTimeBetweenSessions.rawValue] as? Int {
             appsFlyer.minTimeBetweenSessions = UInt(minTimeBetweenSessions)
         }
-        if let anonymizeUser = settings[AppsFlyerConstants.Configuration.anonymizeUser] as? Bool {
+        if let anonymizeUser = settings[AppsFlyerConstants.Configuration.anonymizeUser.rawValue] as? Bool {
             appsFlyer.anonymizeUser = anonymizeUser
         }
-        if let shouldCollectDeviceName = settings[AppsFlyerConstants.Configuration.collectDeviceName] as? Bool {
+        if let shouldCollectDeviceName = settings[AppsFlyerConstants.Configuration.collectDeviceName.rawValue] as? Bool {
             appsFlyer.shouldCollectDeviceName = shouldCollectDeviceName
         }
-        if let customData = settings[AppsFlyerConstants.Configuration.customData] as? [AnyHashable: Any] {
+        if let customData = settings[AppsFlyerConstants.Configuration.customData.rawValue] as? [AnyHashable: Any] {
             appsFlyer.customData = customData
         }
+        if let useUninstallSandbox = settings[AppsFlyerConstants.Configuration.useUninstallSandbox.rawValue] as? Bool {
+            appsFlyer.useUninstallSandbox = useUninstallSandbox
+        }
+        if let enableTCFDataCollection = settings[AppsFlyerConstants.Configuration.enableTCFDataCollection.rawValue] as? Bool {
+            appsFlyer.enableTCFDataCollection(enableTCFDataCollection)
+        }
+        if let appInviteOneLinkID = settings[AppsFlyerConstants.Configuration.appInviteOneLinkID.rawValue] as? String {
+            appsFlyer.appInviteOneLinkID = appInviteOneLinkID
+        }
+        if let deepLinkTimeout = settings[AppsFlyerConstants.Configuration.deepLinkTimeout.rawValue] as? Int {
+            appsFlyer.deepLinkTimeout = UInt(deepLinkTimeout)
+        }
+        if let oneLinkCustomDomains = settings[AppsFlyerConstants.Configuration.oneLinkCustomDomains.rawValue] as? [String] {
+            appsFlyer.oneLinkCustomDomains = oneLinkCustomDomains
+        }
+        if let useReceiptValidationSandbox = settings[AppsFlyerConstants.Configuration.useReceiptValidationSandbox.rawValue] as? Bool {
+            appsFlyer.useReceiptValidationSandbox = useReceiptValidationSandbox
+        }
+        // Wait for ATT authorization if configured (iOS 14+ only)
+        if let attTimeout = settings[AppsFlyerConstants.Configuration.waitForATTUserAuthorizationTimeoutInterval.rawValue] as? Int {
+            if #available(iOS 14, *) {
+                appsFlyer.waitForATTUserAuthorization(timeoutInterval: TimeInterval(attTimeout))
+            }
```

Comment:
You can directly cast to Double (which is the same as TimeInterval)

```suggestion
        if let attTimeout = settings[AppsFlyerConstants.Configuration.waitForATTUserAuthorizationTimeoutInterval.rawValue] as? Double {
            if #available(iOS 14, *) {
                appsFlyer.waitForATTUserAuthorization(timeoutInterval: attTimeout)
            }
```